srcpkgs/gns3-server/patches/requirements.patch

```diff
@@ -7,7 +7,8 @@ diff --git a/requirements.txt b/requirements.txt
 -aiohttp==3.7.4.post0
 +aiohttp>=3.7.4
  aiohttp-cors==0.7.0
- aiofiles==0.7.0
+-aiofiles==0.7.0
++aiofiles==0.8.0
```

Comment:
What would you think about making this `>=`?

Reply:
I thought about doing that originally.  I pushed a change to use `>=`.